### ISSUE TITLE
Width differences across GRAD masters

### ISSUE THREAD

**Issue by chrissimpkins** [googlefonts/googlesans#252]
We reviewed the advance widths across the GRAD masters to confirm that line length will not change across the GRAD axis.  Nikolaus identified the following list of width differences:

```
UFO GoogleSans-opsz17-wght380-GRAD-50.ufo width mismatches compared to GoogleSans-opsz17-wght380-GRAD0.ufo:
  .notdef
  Hahook-cy
  Yusbig-cy
  __descender-cy
  g.sc.BRACKET.18
  hahook-cy
  lslash.sc
  y.BRACKET.18
  yusbig-cy
UFO GoogleSans-opsz17-wght380-GRAD200.ufo width mismatches compared to GoogleSans-opsz17-wght380-GRAD0.ufo:
  .notdef
  Hahook-cy
  Yusbig-cy
  hahook-cy
  lslash.sc
  yusbig-cy
UFO GoogleSans-opsz18-wght380-GRAD-50.ufo width mismatches compared to GoogleSans-opsz18-wght380-GRAD0.ufo:
  .notdef
  Hahook-cy
  Yusbig-cy
  __descender-cy
  g.sc.BRACKET.18
  hahook-cy
  lslash.sc
  yusbig-cy
UFO GoogleSans-opsz18-wght380-GRAD200.ufo width mismatches compared to GoogleSans-opsz18-wght380-GRAD0.ufo:
  .notdef
  Hahook-cy
  Yusbig-cy
  hahook-cy
  lslash.sc
  y.BRACKET.18
  yusbig-cy
UFO GoogleSansItalic-opsz17-wght380-GRAD-50.ufo width mismatches compared to GoogleSansItalic-opsz17-wght380-GRAD0.ufo:
  Dje-cy
  Gbreve
  Gcircumflex
  Gcommaaccent
  Gdotaccent
  Hahook-cy
  Kahook-cy
  Tshe-cy
  U-cy
  Udieresis-cy
  Uhungarumlaut-cy
  Umacron-cy
  Ushort-cy
  __descender-cy.case
  brevecombcy.sc
  dje-cy.sc
  dzhe-cy.sc
  enghe-cy.sc
  ghemiddlehook-cy.sc
  hahook-cy
  hahook-cy.sc
  kahook-cy
  kahook-cy.sc
  lje-cy.sc
  pertenthousand
  pertenthousand.cap
  shcha-cy.sc
  tshe-cy.sc
  y
  yacute
  ycircumflex
  ydieresis
  ydotbelow
  ygrave
  yhookabove
  ytilde
UFO GoogleSansItalic-opsz17-wght380-GRAD200.ufo width mismatches compared to GoogleSansItalic-opsz17-wght380-GRAD0.ufo:
  Dje-cy
  Gbreve
  Gcircumflex
  Gcommaaccent
  Gdotaccent
  Hahook-cy
  Kahook-cy
  Tshe-cy
  U-cy
  Udieresis-cy
  Uhungarumlaut-cy
  Umacron-cy
  Ushort-cy
  __descender-cy.case
  brevecombcy.sc
  dje-cy.sc
  dzhe-cy.sc
  enghe-cy.sc
  ghemiddlehook-cy.sc
  hahook-cy
  hahook-cy.sc
  kahook-cy
  kahook-cy.sc
  lje-cy.sc
  pertenthousand
  pertenthousand.cap
  shcha-cy.sc
  tshe-cy.sc
  y
  yacute
  ycircumflex
  ydieresis
  ydotbelow
  ygrave
  yhookabove
  ytilde
UFO GoogleSansItalic-opsz18-wght380-GRAD-50.ufo width mismatches compared to GoogleSansItalic-opsz18-wght380-GRAD0.ufo:
  Dje-cy
  Gbreve
  Gcircumflex
  Gcommaaccent
  Gdotaccent
  Hahook-cy
  Kahook-cy
  Tshe-cy
  U-cy
  Udieresis-cy
  Uhungarumlaut-cy
  Umacron-cy
  Ushort-cy
  __descender-cy.case
  brevecombcy.sc
  dje-cy.sc
  dzhe-cy.sc
  enghe-cy.sc
  ghemiddlehook-cy.sc
  hahook-cy
  hahook-cy.sc
  kahook-cy
  kahook-cy.sc
  lje-cy.sc
  pertenthousand
  pertenthousand.cap
  shcha-cy.sc
  tshe-cy.sc
  y
  yacute
  ycircumflex
  ydieresis
  ydotbelow
  ygrave
  yhookabove
  ytilde
UFO GoogleSansItalic-opsz18-wght380-GRAD200.ufo width mismatches compared to GoogleSansItalic-opsz18-wght380-GRAD0.ufo:
  Dje-cy
  Gbreve
  Gcircumflex
  Gcommaaccent
  Gdotaccent
  Hahook-cy
  Kahook-cy
  Tshe-cy
  U-cy
  Udieresis-cy
  Uhungarumlaut-cy
  Umacron-cy
  Ushort-cy
  __descender-cy.case
  brevecombcy.sc
  dje-cy.sc
  dzhe-cy.sc
  enghe-cy.sc
  ghemiddlehook-cy.sc
  hahook-cy
  hahook-cy.sc
  kahook-cy
  kahook-cy.sc
  lje-cy.sc
  pertenthousand
  pertenthousand.cap
  shcha-cy.sc
  tshe-cy.sc
  y
  yacute
  ycircumflex
  ydieresis
  ydotbelow
  ygrave
  yhookabove
  ytilde
```

**Comment by madig** [comment]
Script at https://github.com/googlefonts/googlesans/blob/import-berlow-v2/scripts/find-width-diffs.py

**Comment by davelab6** [comment]
Nice! :) https://github.com/googlefonts/fontbakery/issues/3187

**Comment by dberlow** [comment]
GoogleSans-opsz18-wght380-GRAD200.ufo 
 to GoogleSans-opsz18-wght380-GRAD0.ufo:
>   .notdef
>   Hahook-cy CHECK
>   Yusbig-cy CHECK
>   hahook-cy CHECK
>   lslash.sc CHECK
>   y.BRACKET.18 CHECK
>   yusbig-cy CHECK

One down.

**Comment by dberlow** [comment]
Complete as well and ready for build

UFO GoogleSansItalic-opsz18-wght380-GRAD-50.ufo 
to GoogleSansItalic-opsz18-wght380-GRAD0.ufo:
  Dje-cy
  Gbreve
  Gcircumflex
  Gcommaaccent
  Gdotaccent
  Hahook-cy
  Kahook-cy
  Tshe-cy
  U-cy
  Udieresis-cy
  Uhungarumlaut-cy
  Umacron-cy
  Ushort-cy
  __descender-cy.case
  brevecombcy.sc
  dje-cy.sc
  dzhe-cy.sc
  enghe-cy.sc
  ghemiddlehook-cy.sc
  hahook-cy
  hahook-cy.sc
  kahook-cy
  kahook-cy.sc
  lje-cy.sc
  pertenthousand
  pertenthousand.cap
  shcha-cy.sc
  tshe-cy.sc
  y
  yacute
  ycircumflex
  ydieresis
  ydotbelow
  ygrave
  yhookabove
  ytilde

**Comment by chrissimpkins** [comment]
Thanks David!  Please let us know when you've covered all of the masters and we will pull them all in at once.  I really appreciate it.

@madig I'll pull these commits into my downstream branch as he completes updates

**Comment by dberlow** [comment]
discard comment

**Comment by chrissimpkins** [comment]
Fixed in #260
